src/busco/script.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+## VIASH START
+## VIASH END
+
+
+[[ "$par_tar" == "false" ]] && unset par_tar
+[[ "$par_force" == "false" ]] && unset par_force
+[[ "$par_offline" == "false" ]] && unset par_offline
+[[ "$par_opt_out_run_stats" == "false" ]] && unset par_opt_out_run_stats
+[[ "$par_quiet" == "false" ]] && unset par_quiet
+[[ "$par_restart" == "false" ]] && unset par_restart
+[[ "$par_auto_lineage" == "false" ]] && unset par_auto_lineage
+[[ "$par_auto_lineage_euk" == "false" ]] && unset par_auto_lineage_euk
+[[ "$par_auto_lineage_prok" == "false" ]] && unset par_auto_lineage_prok
+[[ "$par_augustus" == "false" ]] && unset par_augustus
+[[ "$par_long" == "false" ]] && unset par_long
+[[ "$par_scaffold_composition" == "false" ]] && unset par_scaffold_composition
+[[ "$par_miniprot" == "false" ]] && unset par_miniprot
+
+if [[ -n "$par_output_prefix" ]]; then 
+    prefix="$par_output_prefix"
+else 
+    prefix="$(basename -- $par_input)"
+fi
+
+if [[ -n "$meta_cpus" ]]; then 
+    cpus="$meta_cpus"
+else 
+    cpus=1
+fi
+
+busco \
+    --in "$par_input" \
+    --mode "$par_mode" \
+    --cpu "$cpus" \
+    --out "$prefix" \
+    ${par_lineage_dataset:+--lineage_dataset "$par_lineage_dataset"} \
+    ${par_augustus:+--augustus} \
+    ${par_augustus_parameters:+--augustus_parameters "$par_augustus_parameters"} \
+    ${par_augustus_species:+--augustus_species "$par_augustus_species"} \
+    ${par_auto_lineage:+--auto-lineage} \
+    ${par_auto_lineage_euk:+--auto-lineage-euk} \
+    ${par_auto_lineage_prok:+--auto-lineage-prok} \
+    ${par_contig_break:+--contig_break $par_contig_break} \
+    ${par_datasets_version:+--datasets_version "$par_datasets_version"} \
+    ${par_e_value:+--evalue "$par_e_value"} \
+    ${par_force:+--force} \
+    ${par_limit:+--limit "$par_limit"} \
+    ${par_long:+--long} \
+    ${par_metaeuk_parameters:+--metaeuk_parameters "$par_metaeuk_parameters"} \
+    ${par_metaeuk_rerun_parameters:+--metaeuk_rerun_parameters "$par_metaeuk_rerun_parameters"} \
+    ${par_miniprot:+--miniprot} \
+    ${par_offline:+--offline} \
+    ${par_opt_out_run_stats:+--opt-out-run-stats} \
+    ${par_quiet:+--quiet} \
+    ${par_restart:+--restart} \
+    ${par_scaffold_composition:+--scaffold_composition} \
+    ${par_tar:+--tar} \
+    ${par_download_base_url:+--download_base_url "$par_download_base_url"} \
+    ${par_download_path:+--download_path "$par_download_path"} \
+    ${par_download:+--download "$par_download"} 
+
+mkdir $par_output_dir
+mv $prefix/* $par_output_dir
```

Comment:
Why not use `--out_path "${par_output_dir}"` to output to this directory by default?

Reply:
because the "prefix" becomes both the name output folder and a prefix for all the filenames... so I did it this way to keep 'prefix' just as a filename prefix and give a separate output folder for publishing results

Reply:
with slight modifications - wildcard run_* wasn't accepted in the cp command, replaced it with find commands instead